docs/py3_todo.md

```diff
@@ -0,0 +1,29 @@
+#Python 3 TODO
+List of known issues, bugs and todos for stash py3 (and py2) compatibility.
+
+#General
+- ctype threads in py3 bugged (crash on cc and kill)
+- more commands need to be ported
+- sys.argv needs to be bytestr in py2 and unistt in py3. There is only a quick fix in place (somewhere in `shruntime.py`), which should be replaced as it only works for common usage situations
```

Comment:
Would this do the trick?  __args = [six.text_type(arg) for arg in sys.argv]__?

Reply:
@cclauss `six.text_type` is `unicode` in py2, but we need a bytestring.

Reply:
http://six.readthedocs.io/#six.binary_type ?

Reply:
How about `str`?